napari/__main__.py

```diff
@@ -441,6 +441,13 @@ def main():
 
         multiprocessing.set_start_method('fork')
 
+    # Prevent __main__.py from being the app name if run with python -m napari
+    if sys.platform == "darwin" and not sys.argv[0].endswith("/napari"):
+        # we need a path with basename `napari`; adding ../ makes it an existing
+        # path too, just in case -- we undo this later once QApplication() is created
+        sys._old_argv_0 = sys.argv[0]
```

Comment:
@jaimergp can you tell me what `sys.argv[0]` *was* in this case? ie why was the basename *not* napari?

Reply:
Or is this `python`? And if so, why replace it with `/../napari`? (ie I don't see why that is guaranteed to exist)

Reply:
If invoked with `python -m napari`, it is set to `/path/to/installation/napari/__main__.py`. See screenshot in PR description, which is what ends up in the application menu.

Reply:
Adding `/../napari` turns `sys.argv[0]` into `/path/to/installation/napari/__main__.py/../napari`, which should be equivalent to `/path/to/installation/napari`.

Reply:
Ah wait, it should be `/../../napari` in that case...

```python
>>> import os
>>> os.path.normpath("/path/to/installation/napari/__main__.py/../napari")
'/path/to/installation/napari/napari'
>>> os.path.normpath("/path/to/installation/napari/__main__.py/../../napari")
'/path/to/installation/napari
```